Compression_code.py

```diff
@@ -1,5 +1,6 @@
 """This code deals with compressing JPEGs. It´s currently in an early state
-but in the end this package alone will be able to randomly compress JPEG files to any required size."""
+but in the end this package alone will be able to randomly compress JPEG files to any required size.
```

Comment:
I don't understand the "randomly compress" -->  does it mean that this file will contain a function that will be used in the actual experiment?